libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +307,141 @@ export class XCCDFResultsMapper extends BaseConverter {
               description: {
-                path: ['Rule.description.text', 'Rule.description'],
-                transformer: (description: string) =>
-                  JSON.stringify(
-                    _.pickBy(convertEncodedXmlIntoJson(description), _.identity)
-                  )
+                path: ['description.text', 'description'],
+                transformer: (description: string): string => {
+                  return _.get(
+                    convertEncodedXmlIntoJson(description),
+                    'VulnDiscussion',
+                    description
+                  ) as string;
+                }
               },
-              group_id: {path: 'id'},
-              group_title: {path: ['title.text', 'title']},
+              group_id: {path: 'group.id'},
+              group_title: {path: ['group.title.text', 'group.title']},
               group_description: {
-                path: ['description.text', 'description'],
-                transformer: (description: string) =>
-                  JSON.stringify(
-                    _.pickBy(convertEncodedXmlIntoJson(description), _.identity)
-                  )
+                path: ['group.description.text', 'group.description'],
+                transformer: (description: string): string => {
+                  return _.get(
+                    convertEncodedXmlIntoJson(description),
+                    'GroupDescription',
+                    description
+                  ) as string;
+                }
               },
-              rule_id: {path: 'Rule.id'},
-              check: {path: 'Rule.check'},
-              fix_id: {path: 'Rule.fix.id'},
+              rule_id: {path: 'id'},
+              check: {path: 'check'},
+              fix_id: {path: 'fix.id'},
               fixtext_fixref: {
-                path: 'Rule.fixtext.fixref'
+                path: ['fixtext.fixref.text', 'fixtext.fixref']
               },
-              ident: {path: 'Rule.ident'},
-              reference: {path: 'Rule.reference'},
-              selected: {path: 'Rule.selected'},
-              version: {path: 'Rule.version.text'},
-              weight: {path: 'Rule.weight'},
-              profiles: {
-                path: '$.Benchmark.Profile',
-                transformer: (
-                  profiles: Record<string, unknown>[]
-                ): Record<ProfileKey, unknown>[] => {
-                  const pathsSelect = ['select'];
-                  const paths = {
-                    id: ['id'],
-                    description: ['description.text', 'description'],
-                    title: ['title.text', 'title']
-                  };
-                  return getProfiles(profiles, pathsSelect, paths);
+              ident: {path: 'ident'},
+              reference: {path: 'reference'},
+              selected: {path: 'selected'},
+              version: {path: ['id', 'version']}, // dunno why the field is called version when it's just an old id
```

Comment:
i think i had id in here as a bug, so we should be safe to make it only 'version' unless you've found id to also be relevant here.  i fixed it when i did the xml parser update.

Reply:
Removing `id` does change the output resulting in "version" not being set in some examples.

Perhaps we can consider that in a separate issue from this one?

Reply:
Sure we can make this a separate issue.  Maybe we change the order of attempted paths such that it attempts to find 'version' before using 'id' as a fallback?

Reply:
> Maybe we change the order of attempted paths such that it attempts to find 'version' before using 'id' as a fallback?

Sure, I've added a commit that does so.

Reply:
Looks great.  We can probably remove the comment there too :)

Reply:
I've added a commit that remove that comment :)